app/models/organization.rb

```diff
@@ -0,0 +1,7 @@
+class Organization < ActiveRecord::Base
+  validates :name, presence: true
+  validates :name, uniqueness: true
```

Comment:
Lines 2 & 3 could be on a single line: `validates :name, presence: true, uniqueness: true`